persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StructuralVariantMapper.xml

```diff
@@ -178,9 +206,15 @@
         and structural_variant.INTERNAL_ID = alteration_driver_annotation.ALTERATION_EVENT_ID
         <where>
             <include refid="whereInMultipleMolecularProfiles"/>
-            <include refid="whereWithEntrezGeneIds"/>
+            <trim prefix="AND (" suffix=")" prefixOverrides="OR">
+                <include refid="whereWithEntrezGeneIds"/>
+                <include refid="whereStructuralVariantQueries"/>
+            </trim>
         </where>
-        ORDER BY gene1.HUGO_GENE_SYMBOL, gene2.HUGO_GENE_SYMBOL
+        ORDER BY gene1.HUGO_GENE_SYMBOL, gene2.HUGO_GENE_SYMBOLprefix="AND (" suffix=")" prefixOverrides="OR">
```

Comment:
@pvannierop The project build failed, see the error log here: https://github.com/cBioPortal/cbioportal/actions/runs/4111128505/jobs/7094596922.

I think it's because we have some error in this line

Reply:
Fixed this!